Node Adapter server.host: true Issue (#10282)
withastro/astro: .changeset/smooth-singers-kiss.md

---
"@astrojs/node": patch
---

> Modified node package to properly assign server.host option set to `true` to listen on all network interfaces.

Reviewer: ```suggestion
Fixes the `server.host` option to properly listen on all network interfaces when set to `true`
```

> Modified node package to properly assign server.host option set to `true` to listen on all network interfaces. Added test to test this functionality.

Reviewer 2: This is great! It describes perfectly what this PR does.

This changeset would become part of [node changelog](https://github.com/withastro/astro/blob/main/packages/integrations/node/CHANGELOG.md), and the primary reader would be astro users. With that in mind, we would want leave out the statement about tests, as that's between maintainers and adequately communicated with the PR itself.
```suggestion
Modified node package to properly assign server.host option set to `true` to listen on all network interfaces.
```
